ops/e2e_test.go

```diff
@@ -81,7 +81,7 @@ var tests = []struct {
 	},
 }
 
-func TestAddChain_Main(t *testing.T) {
+func TestOps_Main(t *testing.T) {
```

Comment:
Let's change this test name to `TestOps_AddNewChain` since its specifically testing that command. And will align with the naming convention of other tests (i.e. `TestOps_<command-name>`)